node/cn/gasprice/feehistory.go

```diff
@@ -172,33 +172,63 @@ func (oracle *Oracle) processBlock(bf *blockFees, percentiles []float64) {
 // This part has a different implementation with Ethereum.
 // Note: an error is only returned if retrieving the head header has failed. If there are no
 // retrievable blocks in the specified range then zero block count is returned with no error.
-func (oracle *Oracle) resolveBlockRange(ctx context.Context, lastBlock rpc.BlockNumber, blocks int) (uint64, int, error) {
-	var headBlock rpc.BlockNumber
-	// query either pending block or head header and set headBlock
-	if lastBlock == rpc.PendingBlockNumber {
-		// pending block not supported by backend, process until latest block
-		lastBlock = rpc.LatestBlockNumber
-		blocks--
-		if blocks == 0 {
-			return 0, 0, nil
-		}
+func (oracle *Oracle) resolveBlockRange(ctx context.Context, reqEnd rpc.BlockNumber, blocks uint64) (*types.Block, []*types.Receipt, uint64, uint64, error) {
```

Comment:
The comment says this:
```
// Pending block does not exist in Kaia, so there is no logic to look up pending blocks.
// This part has a different implementation with Ethereum.
```

Does this PR remove the difference?

Reply:
Yes, it does. I'll remove the comment.